[2025-03-24T12:35:06.558+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:35:06.715+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-02-02T06:00:00+00:00 [queued]>
[2025-03-24T12:35:06.792+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-02-02T06:00:00+00:00 [queued]>
[2025-03-24T12:35:06.793+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T12:35:06.902+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): ingest> on 2024-02-02 06:00:00+00:00
[2025-03-24T12:35:06.937+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3164) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T12:35:06.958+0000] {standard_task_runner.py:72} INFO - Started process 3203 to run task
[2025-03-24T12:35:06.967+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'ingest', 'scheduled__2024-02-02T06:00:00+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmpbhqn2uvn']
[2025-03-24T12:35:06.970+0000] {standard_task_runner.py:105} INFO - Job 134: Subtask ingest
[2025-03-24T12:35:07.241+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-02-02T06:00:00+00:00 [running]> on host 4791d3610749
[2025-03-24T12:35:07.747+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2024-02-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-02T06:00:00+00:00'
[2025-03-24T12:35:07.753+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:35:07.851+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T12:35:07.856+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "ls"']
[2025-03-24T12:35:07.938+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T12:35:07.973+0000] {subprocess.py:106} INFO - ls
[2025-03-24T12:35:07.974+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T12:35:08.077+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:35:08.078+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=ingest, run_id=scheduled__2024-02-02T06:00:00+00:00, execution_date=20240202T060000, start_date=20250324T123506, end_date=20250324T123508
[2025-03-24T12:35:08.226+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:35:08.290+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-24T12:35:08.299+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-25T11:09:22.560+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T11:09:22.758+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-02-02T06:00:00+00:00 [queued]>
[2025-03-25T11:09:22.913+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-02-02T06:00:00+00:00 [queued]>
[2025-03-25T11:09:22.930+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-25T11:09:23.052+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): ingest> on 2024-02-02 06:00:00+00:00
[2025-03-25T11:09:23.092+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=116) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T11:09:23.110+0000] {standard_task_runner.py:72} INFO - Started process 158 to run task
[2025-03-25T11:09:23.128+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'ingest', 'scheduled__2024-02-02T06:00:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmp7hg_4vg7']
[2025-03-25T11:09:23.170+0000] {standard_task_runner.py:105} INFO - Job 176: Subtask ingest
[2025-03-25T11:09:23.510+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-02-02T06:00:00+00:00 [running]> on host 4c8ab9810a8c
[2025-03-25T11:09:24.152+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2024-02-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-02T06:00:00+00:00'
[2025-03-25T11:09:24.154+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-25T11:09:24.406+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-25T11:09:24.408+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "ls"']
[2025-03-25T11:09:24.527+0000] {subprocess.py:99} INFO - Output:
[2025-03-25T11:09:24.532+0000] {subprocess.py:106} INFO - ls
[2025-03-25T11:09:24.536+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-25T11:09:24.712+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-25T11:09:24.723+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=ingest, run_id=scheduled__2024-02-02T06:00:00+00:00, execution_date=20240202T060000, start_date=20250325T110922, end_date=20250325T110924
[2025-03-25T11:09:24.902+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T11:09:24.989+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-25T11:09:25.016+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
